packages/sirius-web/frontend/sirius-web-application/src/views/project-settings/useViewableTabContribution.types.ts

```diff
@@ -0,0 +1,20 @@
+/*******************************************************************************
+ * Copyright (c) 2025 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+
+import { GQLProject, ProjectSettingTabContribution } from './ProjectSettingsView.types';
+
+export interface UseViewableTabContributionValue {
```

Comment:
This doesn't return just the viewable tabs, it does two things at once which is what we don't want